src/App.js

```diff
@@ -2,14 +2,20 @@
 
 import * as React from 'react';
 import { I18nextProvider } from 'react-i18next';
-import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import ApolloClient from 'apollo-boost';
+import { ApolloProvider } from 'react-apollo';
 import initTranslation from './initTranslation';
+import store from './store';
 import Routes from './Routes';
+import Layout from './Layout';
 
 type Props = {
   locale: Object,
 };
 
+const client = new ApolloClient({ uri: process.env.GRAPHQL_URI });
```

Comment:
Should we maybe add the required `Auth` fields here already? Some end-points require it.

We can also leave this for another issue, and plan it a bit more scalably...

![image](https://user-images.githubusercontent.com/2466887/37979173-2ce23512-31e8-11e8-865e-9f9c2b746e39.png)

Reply:
We shouldn't need it for FAQ queries and if so, we can always add it later

Reply:
ha, you might need it for bookings - then I would do it in PR related to that 🙂

Reply:
Yep, I tried bookings first thing, so that's why it failed. But I agree it's better to move it to another PR, and better plan it. Since we will encounter the case where we initialize the `client` without token, and then, once the user logs-in, will need to re-configure the client with the new token. :)